demos/code/index.html

```diff
@@ -33,8 +33,9 @@
         <script src="Blocks/Chapter 6 - Arrays.js"></script>
         <script src="Blocks/Chapter 6 - Arrays (2D).js"></script>
         <script src="Blocks/Chapter 6 - Vectors.js"></script>
+        <script src="Blocks/Chapter 6 - Pointers.js"></script>
         <script src="Blocks/Chapter 7 - Classes.js"></script>
-        <script src="Blocks/Chapter 8 - Pointers.js"></script>
+        
```

Comment:
Pointers are introduced in Chapter 6...not Chapter 8 as the previous file name would suggest.